### PR TITLE
Hide footer on print.

### PR DESCRIPTION
**Pull request by Natim**
r? @magopian

### CODE REVIEW

**Comment by Natim** [comment]
Do you know of a way to resize the canvas before print?

I tried

```
      // Resize canvas on print
      (function (oldPrint) { 
        window.print = function () {
          console.log("Redraw before print");
          canvas.width = 250;
          canvas.height = canvasWrapper.clientWidth * 0.5625;
          redraw();
          oldPrint();
        }
      })(window.print);
```

Without success so far.

**Comment by magopian** [comment]
I don't think you want to resize the canvas, it should be hidden in favor of the image that displays the signature once you "validated" it.

**Comment by Natim** [comment]
Note that exporting from the mobile works like that, it is when I export from the computer that I have two pages because the signature is too big.

File: index.html [hunk 1]
```diff
@@ -151,6 +151,20 @@
...
+                display: none;
+            }
+
+            #refaire-signature {
+                display: none;
+            }
+
+            #canvas-wrapper {
+                width: 250px;
+            }
```
Comment (Natim) [inline index.html]:
```suggestion
```

**Comment by magopian** [comment]
Thanks a lot @Natim, can you please double check that my last modifications are working correctly for you, even on mobile?

**Comment by Natim** [comment]
Ok

**Comment by Natim** [comment]
image

I don't know if we can hide the date selector on print?

**Comment by magopian** [comment]
You mean https://stackoverflow.com/questions/52506410/hiding-a-clear-button-from-date-picker-on-print-in-firefox ? Seems it's not possible? :/

**Comment by Natim** [comment]
I tested it on mobile it works like a charm.

**Comment by magopian** [comment]
Thanks again for the work @Natim :+1:

**Pull request merged**